pkg/webui/frontend/src/components/ConfigForm.vue

```diff
@@ -0,0 +1,202 @@
+              name="envName"        
+              :placeholder="placeholders.envName"    
+              validation="required"
+            />
+            <FormulateInput
+              name="envValue"                        
+              :placeholder="placeholders.envValue"    
+            />
+          </div>
+      </FormulateInput>       
+      </div>
+      <div>
+         <FormulateInput
+          type="text"
+          label="Runtime Image"
+          validation="required"
+          validation-name="Runtime Image"
+          error-behavior="live"
+          :placeholder="placeholders.runtimeImage"
+          v-model="config.runtimeImage"
+        />
+         <FormulateInput
+          type="text"
+          label="Output Binary Path"
+          validation="required"
+          validation-name="Output Binary"
+          error-behavior="live"
+          :placeholder="placeholders.outputBinary"
+          v-model="config.outputBinary"
+        />       
+        <FormulateInput
+          type="group"
+          name="exposePorts"
+          :repeatable="true"
+          label="Expose Ports"
+          add-label="+ Add Port"        
+          :values="config.exposePorts"
+          >
+        <div class="ports">
+            <FormulateInput            
+              name="port"            
+              validation="required|number|between:1,65535"
+              :placeholder="placeholders.exposePorts"    
+            />
+          </div>
+      </FormulateInput>
+        <FormulateInput
+          type="group"
+          name="networks"
+          :repeatable="true"
+          label="Networks"
+          add-label="+ Add Network"        
+          :values="config.networks"
+        >
+        <div class="networks">
+            <FormulateInput
+              name="network"                        
+              validation="required"
+              :placeholder="placeholders.networks"    
+            />
+          </div>
+      </FormulateInput>
+       <FormulateInput
+          type="checkbox"
+          label="Debugger Enabled"
+          :placeholder="placeholders.debuggerEnabled"
+          v-model="config.debuggerEnabled"
+        />
+        <FormulateInput
+          type="number"
+          label="Debugger Port" 
+          :placeholder="placeholders.debuggerPort"
+          v-model="config.debuggerPort"      
+          v-if="config.debuggerEnabled == true"
+        />
+      </div>
+    </div>
+      <div class="actions">          
+        <FormulateInput type="submit" label="Save" />        
+      <FormulateInput
+        type="button"
+        label="Reset"
+        data-ghost
+        @click="reset"
+      />
+      </div>
+            
+    </FormulateForm>
+  </div>
+</template>
+
+<script>
+import ConfigService from '../services/ConfigService';
+
+export default {
+  props: {
+    location: String,
+  },
+   data () {
+    return {
+      config: {},
+      placeholders: {
```

Comment:
Move this to a consts file, then create a computed property exposing the imported file to the template.
In general it's best to put things that don't change often as computed properties instead of data.
Since data re-evaluated on every render, and computed properties only re-evaluate when the reactive properties referenced inside them are changed.
https://vuejs.org/v2/guide/computed.html